vcloud-blueprint.yaml

```diff
@@ -54,28 +54,118 @@ inputs:
       For 'ondemand' service type, the value of edge_gateway
       is always 'gateway'
 
+  management_network_name:
+    description: >
+      Name of the existing network that's being used for Cloudify management
+    type: string
+
 node_templates:
 
-  nodecellar:
-    type: nodecellar.nodes.NodecellarApplicationModule
-    relationships:
+  ssh_keypair:
+    type: cloudify.vcloud.nodes.KeyPair
+    properties:
+      public_key:
+        user: { get_input: agent_user }
+        key:  { get_input: agent_public_key }
+
+  nodejs_security_group:
+      type: cloudify.vcloud.nodes.SecurityGroup
+      properties:
+          security_group:
+              name: nodejsvcloud_security_group
+              edge_gateway: { get_input: edge_gateway }
+          rules:
+              - source: external
+                destination: internal
+                destination_port: 8080
+                action: allow
+                description: >
+                      http to nodejs node
+                protocol: TCP
+              - source: host
+                destination: any
+                action: allow
+                description: >
+                       backward network connection for nodejs updates
+                protocol: any
+
+  mongod_security_group:
+      type: cloudify.vcloud.nodes.SecurityGroup
+      properties:
+          security_group:
+              name: mongodvcloud_security_group
+              edge_gateway: { get_input: edge_gateway }
+          rules:
+              - source: host
+                destination: any
+                action: allow
+                description: >
+                       backward network connection for mongod updates
+                protocol: any
 
-      ################################
-      # Setting the mongo connection
-      ################################
+  ###########################################################
+  # A floating ip to be attached to the nodejs host, since
+  # eventually we want to be able to access it
+  # from any machine, on any network.
+  ###########################################################
 
-      - type: node_connected_to_mongo
-        target: mongod
+  nodecellar_floatingip:
```

Comment:
We changed this to "nodecellar_ip". From 3.3m5+ all nodecellar blueprints in this repository should have consistent names for shared node_templates.